src/GafferSceneUIModule/LightEditorBinding.cpp

```diff
@@ -173,20 +184,30 @@ class InspectorColumn : public PathColumn
 			ScenePlug::PathScope scope( scenePath->getContext(), &scenePath->names() );
 			scope.setCanceller( canceller );
 
-			Inspector::ConstResultPtr inspectorResult = m_inspector->inspect();
-			if( !inspectorResult )
+			try
 			{
-				return result;
-			}
+				Inspector::ConstResultPtr inspectorResult = m_inspector->inspect();
+				if( !inspectorResult )
+				{
+					return result;
+				}
 
-			result.value = runTimeCast<const IECore::Data>( inspectorResult->value() );
-			/// \todo Should PathModel create a decoration automatically when we
-			/// return a colour for `Role::Value`?
-			result.icon = runTimeCast<const Color3fData>( inspectorResult->value() );
-			result.background = g_sourceTypeColors.at( (int)inspectorResult->sourceType() );
-			if( auto source = inspectorResult->source() )
+				result.value = runTimeCast<const IECore::Data>( inspectorResult->value() );
+				/// \todo Should PathModel create a decoration automatically when we
+				/// return a colour for `Role::Value`?
+				result.icon = runTimeCast<const Color3fData>( inspectorResult->value() );
+				result.background = g_sourceTypeColors.at( (int)inspectorResult->sourceType() );
+				if( auto source = inspectorResult->source() )
+				{
+					result.toolTip = new StringData( "Source : " + source->relativeName( source->ancestor<ScriptNode>() ) );
+				}
+			}
+			catch( const std::exception& e )
 			{
-				result.toolTip = new StringData( "Source : " + source->relativeName( source->ancestor<ScriptNode>() ) );
+				result.icon = new StringData( "errorSmall.png" );
+				result.toolTip = new StringData( e.what() );
+
+				msg( Msg::Error, "LightEditor", e.what() );
```

Comment:
These column types are destined for use all over the place, not just in the LightEditor, so this message isn't particularly future-proof. I wonder if we need to print a message at all though, since we're now representing the error gracefully in the UI?

Reply:
In 3c396c6a36538cf4a96a2a4188013d1909b477dd I moved the logic to `PathModel::Item::updateData()` which also was logging a warning message. I removed that warning on the same logic as not duplicating the warning here.